Readme.md

```diff
@@ -3,21 +3,23 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T826240)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
 
 # Grid for Blazor - Save and load layout information
 
-The DevExpress Blazor [Grid](https://docs.devexpress.com/Blazor/403143/grid) allows you to save layout information between app sessions. This example demonstrates how to save layouts automatically and illustrates how to manually use the Grid’s API. The [Grid.razor](./CS/SaveAndRestoreLayout/SaveAndRestoreLayout/Pages/Grid.razor) page contains two tab pages with Grid components:
+The DevExpress Blazor [Grid](https://docs.devexpress.com/Blazor/403143/grid) allows you to save layout information between application sessions. This example demonstrates two different approaches on how to save and restore grid layout:
 
-* The first Grid handles the [LayoutAutoSaving](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.LayoutAutoSaving) and [LayoutAutoLoading](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.LayoutAutoLoading) events. Current layout information is saved to [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage). User modifications (paging, sorting, grouping, and filtering) are restored automatically once a page reloads.
+* On the [Auto.razor](./CS/SaveAndRestoreLayout/SaveAndRestoreLayout/Pages/Auto.razor) page, the Grid saves the current layout information to the [local storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) once the layout changes. When the page reloads, the Grid restores the saved layout. Use the Grid's [LayoutAutoSaving](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.LayoutAutoSaving) and [LayoutAutoLoading](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.LayoutAutoLoading) events to implement this behavior.

```

Comment:
Can we mention an approach and its purpose at the beginning? And then proceed with an example's implementation details?

For example:

* Handle the Grid's LayoutAutoSaving and LayoutAutoLoading events to automatically save and restore the grid layout.
On the [Auto.razor] page....

* Call the Grid's [SaveLayout] and [LoadLayout] methods to save and restore the grid layout on demand.
On the [API.razor] page...

You can also use more specific verbs ("Handle" and "Call") instead of "Use".

Reply:
Your idea is so good and simple that I do not know why it did not occur to me. Thank you for your comment, I have rewritten the text according to your suggestion.